__tests__/app.test.js

```diff
@@ -87,12 +87,13 @@ describe("news-api", () => {
                     const { article } = body
                     expect(article.author).toEqual(expect.any(String))
                     expect(article.title).toEqual(expect.any(String))
-                    expect(article.article_id).toEqual(expect.any(Number))
+                    expect(article.article_id).toBe(1)
                     expect(article.body).toEqual(expect.any(String))
                     expect(article.topic).toEqual(expect.any(String))
                     expect(article.created_at).toEqual(expect.any(String))
                     expect(article.votes).toEqual(expect.any(Number))
                     expect(article.article_img_url).toEqual(expect.any(String))
+                    expect(article.comment_count).toEqual(expect.any(String))
```

Comment:
This should be it's own test, please! Someone reading our tests should be able to follow that, at the time of writing, our getArticleById endpoint worked exactly as intended. Following on from that, they should be able to see that there was a later need to add additional functionality.